### PR TITLE
Modified language to adhere to IBM style guide

### PR DESCRIPTION
**Pull request by jc-berger**
Fixes #254 Review Language in Authoring Stacks.

Since I am doing a review of all the authoring stacks docs, I'll be making several PRs. I don't want any single PR to be too large, so I'll make a new PR for every several docs that I modify to meet the IBM Style Guide's requirements.

See related PR, #57.

### CODE REVIEW

File: docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc [hunk 1]
```diff
@@ -68,11 +68,11 @@ commands:
 +
 [NOTE]
 
-* A command can have only one action, though you can use `composite` commands to execute several commands either sequentially or in parallel.
+* A command only has one action. Still, you can use `composite` commands to execute several commands either sequentially or in parallel.
 
 . Command group
 +
-A given command can be assigned to one or more groups that represents the nature of the command.  The support groups are: `build`, `run`, `test` and `debug`. For each of the groups, one default command can be defined in each group by specifying the `isDefault` value.
+You can assign a given command to one or more groups that represents the nature of the command.  The support groups are: `build`, `run`, `test` and `debug`. For each of the groups, define one default command in each group by specifying the `isDefault` value.
```
Comment (Srivaralakshmi) [inline docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:75]: support groups are: `build`, `run`, `test` and `debug`.

Suggest: to bullet list them. Improves readability.

Comment (Srivaralakshmi) [inline docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:75]: For each of the groups, define one default command in each group by specifying the `isDefault` value.

Question:

- If we remove "in each group", will the sentence still convey the same meaning?
- In other words, is this word "in each group" important? What is its value-add?

**Review by Srivaralakshmi — COMMENTED** [review]
@jc-berger Nice work! The document looks so much better now :)

Thanks for the opportunity and learnings. I have put in some comments. Otherwise LGTM. PTAL.

**Comment by jc-berger** [comment]
Thanks @Srivaralakshmi :) I applied all your feedback. Great feedback and suggestions!

**Review by elsony — APPROVED** [review]
/lgtm

**Comment by jc-berger** [comment]
Due to rebase issues, I've closed this PR and opened a new one, #66: https://github.com/devfile/docs/pull/66 

#66 contains all the doc changes made here. I'm closing this PR since we'll work on and merge #66 instead.

**Pull request closed without merging**